src/SourceBuild/content/build.proj

```diff
@@ -57,6 +58,82 @@
     <MSBuild Projects="$(RepoProjectsDir)$(RootRepo).proj" Targets="WritePrebuiltUsageData;ReportPrebuiltUsage" />
   </Target>
 
+  <Target Name="DiscoverSymbolsTarballs"
```

Comment:
Is there a reason this needs to be a separate target? There doesn't seem to be multiple independent targets that have a dependency on this. It would seem that this could just be integrated into `ExtractSymbolsTarballs` and things already have dependencies on that so the ItemGroup will naturally be available to the dependencies.

Reply:
`ExtractSymbolsTarballs` is a batching target - it needs to have the ItemGroup (`SymbolsTarball`), on which to iterate, before the target is executed.

https://learn.microsoft.com/en-us/visualstudio/msbuild/msbuild-batching?view=vs-2022#target-batching